site/data/scans/ssti/all.yml

```diff
@@ -2,55 +2,55 @@ section: All URLs
 details:
 - title: 'Jinja2 - Python'
   rule_40012: Pass
-  rule_40026: Pass
+  rule_40026: FAIL
```

Comment:
This is the DOM XSS scan rule, a recent related change was the update of WebDriver add-ons.